test/testStateMachine.js

```diff
@@ -7,7 +7,7 @@ const STATE = require('../SessionStateEnum.js')
 const XETHRU_STATE = require('../SessionStateXethruEnum.js')
```

Comment:
Can you please change the name of this file to `testXeThruStateMachine.js`?

Also, can you please add a corresponding `testInnosentStateMachine.js` file?

Also, do you think we should extend the smoke tests to include cases that use the Innosent radar?